### ISSUE TITLE
[Documentation]: Specify it with the 'stream_name' or 'stream_id' arguments in OpenEphysRecordingInterface

### ISSUE THREAD

**Issue by hemmingswu** [catalystneuro/neuroconv#426]
### What would you like changed or added to the documentation and why?

Hi,

I would like to know where to specify the 'stream_name' or 'stream_id' arguments in OpenEphysRecordingInterface

Here's my code:
folder_path = f"myfolder"
# Change the folder_path to the appropriate location in your system
interface = OpenEphysRecordingInterface(folder_path=folder_path, verbose=True, stream_name='Record Node 101#Neuropix-PXI-110.ProbeA-AP')
# Extract what metadata we can from the source files
metadata = interface.get_metadata()

But I'm still getting the this error:
ValueError: This reader have several streams:
Names: ['Record Node 101#Neuropix-PXI-110.ProbeA-AP', 'Record Node 101#Neuropix-PXI-110.ProbeA-LFP']
IDs: ['0', '1']. Specify it with the 'stream_name' or 'stream_id' arguments

Thanks.

### Do you have any interest in helping write or edit the documentation?

Yes, but I would need guidance.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

**Comment by hemmingswu** [comment]
Hi @CodyCBakerPhD 

thanks, that explains.

yes, there is a structure.oebin in the folder, so sounds like the new binary kind.

I can share with you my recordings if that helps.

Also, just a minor suggestion: why not make it convert all streams by default? I've seen .nwb files with multiple streams too.

**Comment by CodyCBakerPhD** [comment]
> yes, there is a structure.oebin in the folder, so sounds like the new binary kind.

I'll get a PR started on that then and ping you when it's ready to try out.



> I can share with you my recordings if that helps.

That would indeed help quite a bit. Would you be able to generate a very small (< 10 MB) recording of noise or something analagous? If not I can take whatever files you have of that structure and manually stub them down to a reasonable size for hosting on our repository for public examples.




> Also, just a minor suggestion: why not make it convert all streams by default? I've seen .nwb files with multiple streams too.

Good idea!

We are in the process of creating several multi-stream converters; the next release of NeuroConv will kick this off for SpikeGLX, the most popular format to have multiple streams of recordings. OpenEphys will probably be one of the next ones to follow that pattern.

For other formats, a part of the complexity comes from how it's not always easy to tell the difference between _types_ of streams, such as raw acquisition vs. LFP vs. AUX channels. Each requires different treatment during NWB conversion, hence why we have a separate `DataInterface` for each type of stream (for the most part; AUX channels are currently lacking a bit...). As such, it is currently the users responsibility to specify how each stream maps onto each interface, and [then combine them](https://neuroconv.readthedocs.io/en/main/conversion_examples_gallery/combinations/ecephys_pose_estimation.html) into a `NWBConverter` to convert all streams simultaneously.

**Comment by hemmingswu** [comment]
Hi @CodyCBakerPhD 

Thanks!

Zip file here:
https://www.transfernow.net/dl/20230427iKTvckvf

You will find 2 folders in it, both have 2 nodes, one is with single recording and the other with multiple recordings under the same "session".
I usually close open-ephys every time I finish one recording, so there's always just 1 recording in each session, but other people might have a different need or approach.
I also have 2 recording nodes. I know when I give the path directory at the node level, neuroconv will recognize the dat file just fine, but I don't know if can handle multiple nodes. Again, this can be solved manually, but I'm just trying to give you the real world scenario.

The "just convert the whole thing by default" suggestion is coming from a user perspective, because as a neuroscientist, I'm using neuroconv mainly because I want to share the data, and I don't want to spend too much time on it lol.

**Comment by CodyCBakerPhD** [comment]
@hemmingswu Hi there, just wanted to touch base really quickly

The main fix of adding stream control was included in #445 

To use it right now you'd have to install from the current developer branch

```
pip install git+https://github.com/catalystneuro/neuroconv.git@main
```

We're finishing up a chain of important PRs before cutting a new official release; I'll try to remember to alert you when that is ready as well

The OpenEphys subconverter (the multi-stream class) is planned in a follow-up PR, likely not to be included in the next release
